### DISCUSSION TITLE
Unit Status: Degraded

### DISCUSSION THREAD

**Discussion by Lemilletrente1030** [MikaylaFischler/cc-mek-scada#590] [Support]
Hi guys so i was messing around with the emergency coolant and waste processing thing. Then on my main monitor there was a red blink in "Unit # Status" section, on the Degraded box. What happended and do i fix that ? 

I also cannot select Antimatter production in the waste production too.

Thanks you all in advance

**Comment by MikaylaFischler** [comment]
Closing as resolved on discord
